packages/GA4Client/test/src/tests.js

```diff
@@ -1346,7 +1360,6 @@ describe('Google Analytics 4 Event', function () {
                 window.dataLayer[0].should.eql(result);
 
                 done();
-            });
```

Comment:
This should come back. Might have gotten snipped when you fixed the merge conflict.

Reply:
Nice catch.  Updated!